From 134fa28069ca79458feaa129eac5453779e733b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20Faria?= <sergio.faria@is4health.com>
Date: Mon, 9 Jul 2018 13:55:20 +0100
Subject: [PATCH] Disable best quality

---
 src/src_sinc.c | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/src/src_sinc.c b/src/src_sinc.c
index 0e4c31f..0603931 100644
--- a/src/src_sinc.c
+++ b/src/src_sinc.c
@@ -33,7 +33,6 @@ typedef float	coeff_t ;
 
 #include "fastest_coeffs.h"
 #include "mid_qual_coeffs.h"
-#include "high_qual_coeffs.h"
 
 typedef struct
 {	int		sinc_magic_marker ;
@@ -101,10 +100,7 @@ const char*
 sinc_get_name (int src_enum)
 {
 	switch (src_enum)
-	{	case SRC_SINC_BEST_QUALITY :
-			return "Best Sinc Interpolator" ;
-
-		case SRC_SINC_MEDIUM_QUALITY :
+	{	case SRC_SINC_MEDIUM_QUALITY :
 			return "Medium Sinc Interpolator" ;
 
 		case SRC_SINC_FASTEST :
@@ -126,9 +122,6 @@ sinc_get_description (int src_enum)
 		case SRC_SINC_MEDIUM_QUALITY :
 			return "Band limited sinc interpolation, medium quality, 121dB SNR, 90% BW." ;
 
-		case SRC_SINC_BEST_QUALITY :
-			return "Band limited sinc interpolation, best quality, 144dB SNR, 96% BW." ;
-
 		default :
 			break ;
 		} ;
@@ -197,12 +190,6 @@ sinc_set_converter (SRC_PRIVATE *psrc, int src_enum)
 				temp_filter.index_inc = slow_mid_qual_coeffs.increment ;
 				break ;
 
-		case SRC_SINC_BEST_QUALITY :
-				temp_filter.coeffs = slow_high_qual_coeffs.coeffs ;
-				temp_filter.coeff_half_len = ARRAY_LEN (slow_high_qual_coeffs.coeffs) - 2 ;
-				temp_filter.index_inc = slow_high_qual_coeffs.increment ;
-				break ;
-
 		default :
 				return SRC_ERR_BAD_CONVERTER ;
 		} ;
-- 
2.17.1

